app/src/main/res/values/strings.xml

```diff
@@ -1,20 +1,37 @@
-<resources>
+<resources xmlns:tools="http://schemas.android.com/tools">
     <!-- App Default -->
     <string name="app_name">iAnime</string>
     <string name="nav_home_title">Gallery</string>
     <string name="nav_user_title">Mine</string>
     <string name="setting_chinese">CN</string>
     <string name="setting_english">EN</string>
+    <string name="change_zh_message">App language changed to Chinese</string>
+    <string name="change_en_message">App language changed to English</string>
     <string name="user_screen_title" translatable="false">COMING SOON...</string>
 
     <!-- Anime Detail Screen -->
     <string name="intro_header_title">About</string>
     <string name="anime_type_title">Type: %1$s</string>
     <string name="anime_country_title">Country: %1$s</string>
-    <string name="anime_year_title">Published Year: %1$s</string>
+    <string name="anime_release_year_title">Published Year: %1$s</string>
     <string name="anime_status_title">Status: %1$s</string>
     <string name="maxim_header_title">Maxim</string>
     <string name="episode_header_title">Favourite Episodes</string>
+    <string name="empty_description_message" tools:ignore="TypographyEllipsis">We need you input some more information for this anime...</string>
```

Comment:
you can directly use this: `We need you to provide more information about this anime...`